### PR TITLE
Cleanup on complete

### PR DESCRIPTION
**Pull request by g4s8**
#182 - cleanup old metadata file on batchUpdate complete

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#184](https://github.com/artipie/rpm-adapter/issues/184) is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/d24cf08f-44da-4fcd-8654-9a6b7cb56ec0); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/com/artipie/rpm/RpmITCase.java [hunk 1]
```diff
@@ -51,55 +54,97 @@
  *  and TYPE is a type of file (primary, others, filelists). Don't forget to
  *  uncompress it first.
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle MagicNumberCheck (500 lines)
  */
-@EnabledIfSystemProperty(named = "it.longtests.enabled", matches = "true")
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+//@EnabledIfSystemProperty(named = "it.longtests.enabled", matches = "true")
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/rpm/RpmITCase.java:60]: @g4s8 uncomment this property please

**Comment by g4s8** [comment]
@olenagerasimova thanks, fixed

**Comment by 0crat** [comment]
@olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Pull request merged**